Add basic explanation to ProgrammableAndroids section. (#106)
Slimefun/Wiki: pages/Miner-Androids.md

> A specialized android for mining purposes, but can used as breaking other blocks too.

Reviewer: ```suggestion
An Android specialized for mining purposes, but can used to break other blocks too.
```

## Obtaining
Miner Androids can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).

## Usage

> This android can be used to create tunnel, caves, and others.

Reviewer: ```suggestion
This android can be used to create tunnels, caves, and alikes.
```

## Tiers
There is only one tier in this miner android.

> | Tier | Fuel Efficiency |
> | ---- | --------------- |
> | Basic | 1.0x |

Reviewer: ```suggestion
| Tier  | Fuel Efficiency |
| ----- | :-------------: |
| Basic | 1.0x            |
```
